bin/newman.js

```diff
@@ -34,6 +34,8 @@ program
     .option('--env-var <value>',
         'Allows the specification of environment variables via the command line, in a key=value format',
         util.cast.memoizeKeyVal, [])
+    .option('--ignore-iteration <n>',
```

Comment:
Not sure if `ignore-iteration` is the correct option name. It feels like this should ignore the given iteration number. ​😅